### PR TITLE
docs: update links to learn

### PR DESCRIPTION
**Pull request by alexyarosh**
Redirects mess up source attribution for page views. I'd like to know when people are going from docs to learn, but I can't because every redirect is recorded as having come from the old link for the same page. Since we changed every link in Learn during the redesign, this makes source analysis largely impossible.

Single it's just a few mins of find-and-replace work to update the links, and it enables analysis, I figured why not just update all the links to Learn to their current state.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ Run as a JAR, using Docker, or on [Metabase Cloud](https://store.metabase.com/ch
 
 Once installed, set up your Metabase and connect to your data.
 
-### [Getting started](https://www.metabase.com/learn/getting-started/getting-started)
+### [Getting started](https://www.metabase.com/learn/metabase-basics/getting-started/index)
```
Comment (jeff-bruemmer) [inline docs/README.md:31]:
```suggestion
### [Getting started](https://www.metabase.com/learn/metabase-basics/getting-started)
```
Do we need the index?

File: docs/exploration-and-organization/events-and-timelines.md [hunk 2]
```diff
@@ -124,4 +124,4 @@ Note that for questions outside of that collection, you can only temporarily app
 
 ## Further reading
 
-- [Keeping your Metabase organized](https://www.metabase.com/learn/administration/same-page)
+- [Keeping your Metabase organized](hhttps://www.metabase.com/learn/metabase-basics/administration/administration-and-operation/same-page)
```
Comment (jeff-bruemmer) [inline docs/exploration-and-organization/events-and-timelines.md:127]:
```suggestion
- [Keeping your Metabase organized](https://www.metabase.com/learn/metabase-basics/administration/administration-and-operation/same-page)
```

**Review by jeff-bruemmer — APPROVED** [review]
Might need to run prettier on all of these files, as I see some formatting issues.

File: docs/README.md (hunk 1 above)
Comment (alexyarosh) [reply docs/README.md:31]: I just copy-pasted the resolved link from the address bar. It'll work without the index but I just want to reduce redirects as much as possible

**Pull request merged**